README.md

```diff
@@ -80,9 +80,11 @@ For more information, see the [CoreOS security disclosure page](https://coreos.c
 
 ## Known issues
 
-Due to a bug in the Linux kernel, using rkt's overlay support on top of an overlay filesystem requires Linux 4.3+ (see the [stacked overlay fix](https://github.com/torvalds/linux/commit/1c8a47d) and [#1537](https://github.com/coreos/rkt/issues/1537)).
+Due to limitations in the Linux kernel, using rkt's overlay support on top of an overlay filesystem requires the upperdir, and workdir to support the creation of trusted.* extended attributes, and valid d_type in readdir responses (see [kernel/Documentation/filesystems/overlayfs.txt](https://www.kernel.org/doc/Documentation/filesystems/overlayfs.txt)). When starting rkt inside rkt this means that either:
```

Comment:
I always feel uncomfortable with your use of commas, so I'm usually refraining from commenting on those, but I would naturally have expected this to read `requires the upperdir and workdir to support the creation of trusted.* extended attributes, and valid d_type in readdir responses`.